crates/router/tests/connectors/connector_auth.rs

```diff
@@ -0,0 +1,147 @@
+use std::env;
```

Comment:
This file is moved to crates/test_utils/src/connector_auth.rs. Please remove this new file